src/BenchmarkDotNet/ConsoleArguments/CommandLineOptions.cs

```diff
@@ -218,6 +218,9 @@ public bool UseDisassemblyDiagnoser
         [Option("noForcedGCs", Required = false, HelpText = "Specifying would not forcefully induce any GCs.")]
         public bool NoForcedGCs { get; set; }
 
+        [Option("noEvaluationOverhead", Required = false, HelpText = "Specifying would not run the evaluation overhead iterations.")]
```

Comment:
subjective opinion: `noOverheadEvaluation` sounds more intuitive than `noEvaluationOverhead`. What do you think?

```suggestion
        [Option("noOverheadEvaluation", Required = false, HelpText = "Specifying would not run the evaluation overhead iterations.")]
```